docs/internals/howto-release-django.txt

```diff
@@ -52,22 +55,40 @@ You'll need a few things before getting started:
 * A GPG key. If the key you want to use is not your default signing key, you'll
   need to add ``-u you@example.com`` to every GPG signing command below, where
   ``you@example.com`` is the email address associated with the key you want to
-  use.
+  use. You will also need to add ``-i you@example.com`` to the ``twine``
+  shell invocation.
 
 * An install of some required Python packages:
 
   .. code-block:: shell
 
       $ python -m pip install wheel twine
 
-* Access to Django's record on PyPI. Create a file with your credentials:
+* Access to Django's project on PyPI. Create a project-scoped token following
+  the `official documentation <https://pypi.org/help/#apitoken>`_. At the time
+  of this writing, that means:
+
+  #. Create a new token in your
+     `account settings page <https://pypi.org/manage/account/token/>`_.
+
+  #. Set up your ``$HOME/.pypirc`` file like this:
+
+     .. code-block:: ini
+        :caption: ``~/.pypirc``
+
+        [distutils]
+          index-servers =
+            pypi
+            PROJECT_NAME
 
-  .. code-block:: ini
-    :caption: ``~/.pypirc``
+        [pypi]
+          username = __token__
+          password = # either a user-scoped token or a project-scoped token you want to set as the default
 
-    [pypi]
-    username:YourUsername
-    password:YourPassword
+        [PROJECT_NAME]
+          repository = https://upload.pypi.org/legacy/
+          username = __token__
+          password = # a project token 
```

Comment:
This an instruction for us not for other projects, so I'd change `PROJECT_NAME` to `django`.